### PR TITLE
Add various bindings for creating window icons

### PR DESCRIPTION
**Pull request by georgefst**
I'm building a tiny cross-platform library for setting window icons (hooking up to the JuicyPixels library to make it easy to e.g. use a PNG image). This PR adds everything I required on Windows.

### CODE REVIEW

File: Graphics/Win32/Message.hsc [hunk 1]
```diff
@@ -158,6 +158,7 @@ type WindowMessage   = DWORD
  , wM_QUEUESYNC         = WM_QUEUESYNC
  , wM_USER              = WM_USER
  , wM_APP               = WM_APP
+ , wM_SETICON           = WM_SETICON
```
Comment (georgefst) [inline Graphics/Win32/Message.hsc:161]: Is there any rationale to the ordering here?

Comment (Mistuke) [reply Graphics/Win32/Message.hsc:161]: Not that I'm aware off, they were added from an import from another project and I didn't clean up all the code :)

In principle they should be alphabetical but I'm not that bothered

**Comment by Mistuke** [comment]
Thanks!, this looks good to me, could you add a changelog entry and then it'll be ready to go.

**Comment by georgefst** [comment]
> Thanks!, this looks good to me, could you add a changelog entry and then it'll be ready to go.

Done.

**Comment by Mistuke** [comment]
Cheers

**Pull request merged**